src/components/Common.js

```diff
@@ -42,4 +43,37 @@ export const SearchBar = styled.input.attrs({
   margin: 8px 0;
   border: none;
   color: ${p => p.theme.colors.text};
+`
+
+export const DetailContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+
+  @media (max-width: 1200px) {
+    display: block;
+  }
+`
+
+export const DetailColumn = styled.ul`
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100%;
+  flex: 1;
```

Comment:
```suggestion
  flex: 1 100%;
```
ehehhH https://developer.mozilla.org/en-US/docs/Web/CSS/flex

Reply:
mmm this doesn't work for some reason, i think it's because when flex takes 2 variables it doesn't treat the second one as flex-basis
![image](https://user-images.githubusercontent.com/47487758/97635914-6cebb280-19f5-11eb-8ded-63651a67f242.png)

Reply:
we don't want this, right?
![image](https://user-images.githubusercontent.com/47487758/97635975-87259080-19f5-11eb-86dc-07097a038445.png)

Reply:
Oh, maybe flex basis 100% is getting overwritten by the shorthand right after it?

>When omitted from the flex shorthand, its specified value is the length zero

https://css-tricks.com/almanac/properties/f/flex-basis/

Reply:
So you're saying you want it like this? 
![image](https://user-images.githubusercontent.com/47487758/97651160-91a55180-1a18-11eb-97ae-10f9b5fdd4e0.png)
That's what I had which made the screenshot above, but I might be misunderstanding you

Reply:
done! it just says `flex: 1` now